docs/arlas-tagger-configuration.md

```diff
@@ -71,12 +71,12 @@ docker run -ti -d \
 
 | Environment variable      | ARLAS Tagger configuration variable | Default | Description |
 | --- | --- | --- | ---  |
-| ARLAS_ELASTIC_NODES       | elastic_configuration.elastic_nodes    | localhost:9300 | comma separated list of elasticsearch nodes as host:port values |
+| ARLAS_ELASTIC_NODES       | elastic_configuration.elastic_nodes    | localhost:9200 | comma separated list of elasticsearch nodes as host:port values |
 | ARLAS_ELASTIC_SNIFFING    | elastic_configuration.elastic_sniffing | false          | allow elasticsearch to dynamically add new hosts and remove old ones (*)|
 | ARLAS_ELASTIC_CLUSTER     | elastic_configuration.elastic_cluster  | elasticsearch  | clustername of the elasticsearch cluster that is used for storing ARLAS configuration |
 | ARLAS_ELASTIC_ENABLE_SSL  | elastic-enable-ssl  | false          | use SSL to connect to elasticsearch |
 | ARLAS_ELASTIC_CREDENTIALS | elastic-credentials | user:password  | credentials to connect to elasticsearch |
-| ARLAS_ELASTIC_COMPRESS    | elastic-compress    | true           | transport.compress parameter of the elasticsearch connection |
+| ARLAS_ELASTIC_SKIP_MASTER | elastic-skip-master | true           | Skip dedicated master in Rest client |
 
```

Comment:
I think we should mark `ARLAS_ELASTIC_CLUSTER` as deprecated

Reply:
ok